activation/activation_test.go

```diff
@@ -1195,53 +1135,6 @@ func TestBuilder_PublishActivationTx_FailsWhenNIPostBuilderFails(t *testing.T) {
 	require.NotNil(t, challenge)
 }
 
-func TestBuilder_PublishActivationTx_Serialize(t *testing.T) {
```

Comment:
:bulb: This test was removed as it didn't check anything useful. Testing if the (automatically-generated) scale-encoding/decoding of the wire type works is not the responsibility of Atx Builder tests.